<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DICTDialogConfig</class>
 <widget class="QDialog" name="DICTDialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DICT</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>690</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>991</width>
     <height>661</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Informations</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>966</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Coordonnées</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>29</y>
        <width>78</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dénomination</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coordDenom">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>52</y>
        <width>471</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Personne</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coordPersonne">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>53</y>
        <width>461</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>77</y>
        <width>87</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro et voie</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coordNumVoie">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>581</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="coordBP">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>100</y>
        <width>341</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>77</y>
        <width>65</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lieu Dit /BP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>130</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Commune</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coordCommune">
      <property name="geometry">
       <rect>
        <x>488</x>
        <y>150</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>127</y>
        <width>67</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Code Postal</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coordCP">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>150</y>
        <width>462</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>177</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Téléphone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coordTel">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>200</y>
        <width>261</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>177</y>
        <width>20</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fax</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coordFax">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>200</y>
        <width>321</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>961</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Responsable du dossier</string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>29</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="respNom">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>52</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>27</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Service</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="respService">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>27</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Téléphone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="respTel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>961</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Signataire</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>29</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signNom">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>52</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Signature</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signSignature">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>53</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>961</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Informations relatives à la dégradation des ouvrages</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>29</y>
        <width>123</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro de téléphone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TelEndommagement">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>52</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>27</y>
        <width>95</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro du SDIS</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Endommagement">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Configuration</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Fusion</string>
       </property>
       <widget class="QCheckBox" name="fusionPDF">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>454</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Souhaitez-vous fusionner les documents (plans et récépissé) pour l'envoi ?</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Formatage des noms des fichiers en sortie</string>
       </property>
       <widget class="QLineEdit" name="sufRecep">
        <property name="geometry">
         <rect>
          <x>267</x>
          <y>52</y>
          <width>137</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>29</y>
          <width>57</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Récépissé</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>52</y>
          <width>48</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Préfixe :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prefRecep">
        <property name="geometry">
         <rect>
          <x>68</x>
          <y>52</y>
          <width>137</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>212</x>
          <y>52</y>
          <width>48</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Suffixe :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prefPlan">
        <property name="geometry">
         <rect>
          <x>66</x>
          <y>100</y>
          <width>137</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_31">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>100</y>
          <width>48</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Suffixe :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="sufPlan">
        <property name="geometry">
         <rect>
          <x>265</x>
          <y>100</y>
          <width>137</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>77</y>
          <width>24</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Plan</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>100</y>
          <width>48</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Préfixe :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>137</y>
          <width>401</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Répertoire des fichiers d'information  à annexer à la réponse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="configAnnexes">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>871</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButtonAnnexes">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>160</y>
          <width>27</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>Dossiers</string>
       </property>
       <widget class="QLineEdit" name="configRep">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>861</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>87</y>
          <width>265</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Répertoire de sortie du plan et du formulaire :</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>110</y>
          <width>27</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>137</y>
          <width>278</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Répertoire des gabarits de mise en page (.qpt) :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="configQPT">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>861</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButtonQPT">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>160</y>
          <width>27</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>29</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Répertoire d'importation des fichiers xml :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="configXML">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>861</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButtonXML">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>50</y>
          <width>27</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Stockage BD</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxDbStorage">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>329</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Stocker les demandes/réponses en base de données</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxDbStorage">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>38</y>
       <width>547</width>
       <height>219</height>
      </rect>
     </property>
     <property name="title">
      <string>BD de stockage</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QLabel" name="label_23">
        <property name="minimumSize">
         <size>
          <width>66</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Type de base</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QTabWidget" name="databaseSpecificOptions">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>PostGIS</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QLabel" name="label_20">
            <property name="minimumSize">
             <size>
              <width>39</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Schéma</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEditDbSchemaName">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxDbDictSchema">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>19</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButtonCreateDbSchema">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>créer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Spatialite</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Choisissez une connexion dans la liste ci-dessus, ou</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonCreateSpatialiteDb">
            <property name="text">
             <string>Créer une base Spatialite</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_22">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Connexion</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxDbType">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>19</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>-- Choisir --</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Postgis</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spatialite</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="comboBoxDbConnection">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>19</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>coordDenom</tabstop>
  <tabstop>coordPersonne</tabstop>
  <tabstop>coordNumVoie</tabstop>
  <tabstop>coordBP</tabstop>
  <tabstop>coordCP</tabstop>
  <tabstop>coordCommune</tabstop>
  <tabstop>coordTel</tabstop>
  <tabstop>coordFax</tabstop>
  <tabstop>respNom</tabstop>
  <tabstop>respService</tabstop>
  <tabstop>respTel</tabstop>
  <tabstop>signNom</tabstop>
  <tabstop>signSignature</tabstop>
  <tabstop>TelEndommagement</tabstop>
  <tabstop>Endommagement</tabstop>
  <tabstop>configXML</tabstop>
  <tabstop>toolButtonXML</tabstop>
  <tabstop>configAnnexes</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>DICTDialogConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>DICTDialogConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
